packages/api/endpoints/distribution.js

```diff
@@ -53,15 +51,15 @@ function handler(event, context, cb) {
   // code means that this is a redirect back from
   // earthData login
   if (query.code) {
-    const form = new FormData();
```

Comment:
The `got` module has support for forms built in, so we don't need to use the separate `form-data` module.

Reply:
👍

Reply:
👍